bundles/ch.elexis.core.test.context/.polyglot.META-INF

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
datei löschen - polyglots werden dynamisch erzeugt, eigentlich sollte das in `.gitignore` sein